rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @book.rentals << self
```

Comment:
- [ ] The rental should be added to the original `book` that was passed, not the `@book` as the latter is a local variable of sorts and is not same as the initial one. You just need to remove `@` from line 7

  ```
  book.rentals << self
  ```